services/galley/src/Galley/API/Internal.hs

```diff
@@ -158,6 +158,20 @@ type InternalAPI =
                       :> ReqBody '[Servant.JSON] UpsertOne2OneConversationRequest
                       :> Post '[Servant.JSON] UpsertOne2OneConversationResponse
                   )
+           :<|> Named
+                  "feature-config-snd-factor-password-challenge"
+                  ( Summary "Get the server wide feature config for the 2nd factor password challenge feature."
```

Comment:
```suggestion
                  -- FUTUREWORK: Introduce `/i/feature-configs` and drop this one again.  The internal end-poins has the
                  -- same handler as the public one, plus optional user id in the query.  Maybe require `DoAuth` to disable
                  -- access control only on the internal end-point, not on the public one.  (This may also be a good oppportunity
                  -- to make `AllFeatureConfigs` more type-safe.)
                  ( Summary "Get feature config for the 2nd factor password challenge feature (for user/team; if n/a fall back to site config)."
```